### ISSUE TITLE
MenuItem disabled prop

### ISSUE THREAD

**Issue by albertfdp** [davidtheclark/react-aria-menubutton#67]
Not sure if this is currently possible, but I have the use case of having one item in the Menu being shown but being disabled (`aria-disabled`), it cannot be selected and focus is skipped when navigating with arrow keys.

Happy to contribute with a PR.

**Comment by davidtheclark** [comment]
How about rendering it something other than a `MenuItem` when it is not active?

**Comment by gouegd** [comment]
@davidtheclark I thought at first that you meant adding something like a `MenuLabel` that would not take the focus, contrary to `MenuItem`.
But it is indeed already possible to put anything alongside `MenuItems`. I just tested by adding a div, it gets displayed, does not mess with accessibility (it's skipped by up/down arrows), and does not have the hover effect `MenuItem`s have.
It does still close the menu if you click on it, but without invoking the callback, so maybe it's as it should be.

Great stuff @davidtheclark :)

**Comment by davidtheclark** [comment]
@gouegd: The fact that the menu closes when you click the disabled button is this bug: https://github.com/davidtheclark/react-aria-menubutton/issues/62. Besides that, seems like we can close this issue.
